src/accelerate/commands/config/cluster.py

```diff
@@ -155,177 +151,141 @@ def get_cluster_input():
                         )
+            config["deepspeed_config"] = deepspeed_config
 
-    fsdp_config = {}
-    if distributed_type in [DistributedType.MULTI_GPU]:
+    if config["distributed_type"] in [DistributedType.MULTI_GPU]:
         use_fsdp = _ask_field(
-            "Do you want to use FullyShardedDataParallel? [yes/NO]: ",
-            _convert_yes_no_to_bool,
-            default=False,
-            error_message="Please enter yes or no.",
+            "Do you want to use FullyShardedDataParallel?",
+            "bool",
         )
         if use_fsdp:
-            distributed_type = DistributedType.FSDP
-        if distributed_type == DistributedType.FSDP:
-            sharding_strategy_query = "What should be your sharding strategy ("
+            fsdp_config = {}
+            config["distributed_type"] = DistributedType.FSDP
+        if config["distributed_type"] == DistributedType.FSDP:
+            sharding_strategy_query = "What should be your sharding strategy? ("
             for i, strategy in enumerate(FSDP_SHARDING_STRATEGY):
-                sharding_strategy_query += f"[{i+1}] {strategy}, "
-            sharding_strategy_query = sharding_strategy_query[:-2] + ")? [1]: "
+                sharding_strategy_query += f"[{RICH_COLORS[i]}][{i+1}] {strategy}[/{RICH_COLORS[i]}], "
+            sharding_strategy_query = sharding_strategy_query[:-2] + ")"
             fsdp_config["fsdp_sharding_strategy"] = _ask_field(
                 sharding_strategy_query,
-                lambda x: int(x),
+                "int",
                 default=1,
             )
             fsdp_config["fsdp_offload_params"] = _ask_field(
-                "Do you want to offload parameters and gradients to CPU? [yes/NO]: ",
-                _convert_yes_no_to_bool,
-                default=False,
-                error_message="Please enter yes or no.",
+                "Do you want to offload parameters and gradients to CPU?",
+                "bool",
             )
-            fsdp_wrap_query = "What should be your auto wrap policy ("
+            fsdp_wrap_query = "What should be your auto wrap policy? ("
             for i, wrap_policy in enumerate(FSDP_AUTO_WRAP_POLICY):
-                fsdp_wrap_query += f"[{i}] {wrap_policy}, "
-            fsdp_wrap_query = fsdp_wrap_query[:-2] + ")? [0]: "
-            fsdp_config["fsdp_auto_wrap_policy"] = _ask_field(
-                fsdp_wrap_query,
-                lambda x: FSDP_AUTO_WRAP_POLICY[int(x)],
-                default="TRANSFORMER_BASED_WRAP",
-            )
+                fsdp_wrap_query += f"[{RICH_COLORS[i]}][{i}]{wrap_policy}[/{RICH_COLORS[i]}], "
+            fsdp_wrap_query = fsdp_wrap_query[:-2] + ")"
+            fsdp_config["fsdp_auto_wrap_policy"] = FSDP_AUTO_WRAP_POLICY[
+                _ask_field(
+                    fsdp_wrap_query,
+                    "int",
+                    choices=[*map(str, range(3))],
+                )
```

Comment:
```suggestion
                default=0,
                )
```

Reply:
Getting error when just pressing just enter after seeing (0) as default:
```
/home/sourab/accelerate/src/accelerate/commands/config/cluster.py:214 in get_cluster_input       │
│                                                                                                  │
│   211 │   │   │   for i, wrap_policy in enumerate(FSDP_AUTO_WRAP_POLICY):                        │
│   212 │   │   │   │   fsdp_wrap_query += f"[{RICH_COLORS[i]}][{i}]{wrap_policy}[/{RICH_COLORS[   │
│   213 │   │   │   fsdp_wrap_query = fsdp_wrap_query[:-2] + ")"                                   │
│ ❱ 214 │   │   │   fsdp_config["fsdp_auto_wrap_policy"] = FSDP_AUTO_WRAP_POLICY[                  │
│   215 │   │   │   │   _ask_field(                                                                │
│   216 │   │   │   │   │   fsdp_wrap_query,                                                       │
│   217 │   │   │   │   │   "int",                                                                 │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: list indices must be integers or slices, not str
```

Reply:
Can you try rerunning these again? There's a handler to set the default to `0` and it was a str instead of an int.